app/models/manageiq/providers/embedded_terraform/automation_manager/stack.rb

```diff
@@ -54,4 +54,58 @@ def refresh
   def raw_status
     Status.new(miq_task)
   end
+
+  # Intend to be called by UI to display stdout. The stdout is stored in TerraformRunner(api/stack#message)
+  def raw_stdout_via_worker(userid, format = 'txt')
+    unless MiqRegion.my_region.role_active?("embedded_terraform")
+      msg = "Cannot get standard output of this terraform-template because the embedded terraform role is not enabled"
+      return MiqTask.create(
+        :name    => 'terraform_stdout',
+        :userid  => userid || 'system',
+        :state   => MiqTask::STATE_FINISHED,
+        :status  => MiqTask::STATUS_ERROR,
+        :message => msg
+      ).id
+    end
+
+    options = {:userid => userid || 'system', :action => 'terraform_stdout'}
+    queue_options = {
+      :class_name  => self.class,
+      :method_name => 'raw_stdout',
+      :instance_id => id,
+      :args        => [format],
+      :priority    => MiqQueue::HIGH_PRIORITY,
+      :role        => nil
+    }
+
+    MiqTask.generic_action_with_callback(options, queue_options)
+  end
+
+  def raw_stdout(format = 'txt')
+    case format
+    when "json" then raw_stdout_json
+    when "html" then raw_stdout_html
+    else             raw_stdout_txt
+    end
+  end
+
+  def raw_stdout_json
+    raw_stdout_txt.split("\n")
```

Comment:
This feels like a copy-paste from the ansible one where the raw_stdout_txt is actually json (we ask ansible to give us the raw output in json format, then we massage it later).  In your case, I think it's just regular text, so this method would not be correct.  I don't think we use the json anywhere, so you can either delete the json paths, or raise a NotImplementError here

Reply:
text which is,  terraform stdout with terminal color codes.

yes, json format is not required,  was not sure, if it required in ManageIQ anywhere ...  
As suggested, will raise the raise NotImplementedError,

Reply:
Deleting the json path